ansible/todoapp.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Todo App
+
+[Service]
+User=root
+WorkingDirectory=/opt/todoapp
+ExecStart=/home/ec2-user/.local/bin/poetry run flask run -h 0.0.0.0 -p 80

```

Comment:
Using the default HTTP port (80) means you don't need to enter the port in the browser 👍 
Note that it meant you had to run as root (which is not ideal) because it is a protected port. 

The alternative is to use a systemd "socket" to listen to port 80 and still run the app itself as the ec2-user. No need to do so, I'm just pointing it out.